src/vue-resource-case-converter.js

```diff
@@ -46,42 +54,50 @@ function convertObjectKeys(obj, keyConversionFun) {
   }, Array.isArray(obj) ? [] : {}); // preserve "arrayness"
 }
 
+function mergeOptions(globalOptions, localOptions) {
+  // Combines the value of requestUrlFilter/responseUrlFilter with the global options
+  if (typeof(localOptions) === 'boolean') {
+    return Object.assign({}, globalOptions, {convert: localOptions});
```

Comment:
Thanks, I'm not used to supporting IE8 these days. What's your preference for how to accommodate this? 

1. Babel has an [`Object.assign transform`](https://babeljs.io/docs/plugins/transform-object-assign/) plugin, which I believe you could add into your Gulp pipeline to automatically inline a polyfill. 
2. I could add it as a dependency in package.json, then require it into the main script. 
3. I could inline the whole polyfill directly in the main `src/vue-resource-case-converter.js` file. 

My leaning is for option 1, since it leaves the main script as cleaner code and doesn't require users of the library to install any dependencies. But happy to defer to your preference.